opennms-doc/releasenotes/src/asciidoc/upgrade/upgrade-20-21.adoc

```diff
@@ -0,0 +1,45 @@
+[[upgrade-19-20]]
+== Upgrade OpenNMS Horizon 19 to 20
+
+=== Foreign Sources and Requisitions
+
+The Foreign Sources and Requisitions have been moved to the database.
+They must be manually migrated. In order to make the migration as smooth as possible a `foreignsource` and `requisition` osgi karaf command has been introduced.
```

Comment:
What is the expected behavior if in import happens to triggered before the migration is completed?